### PR TITLE
Better default hooks

### PR DESCRIPTION
**Pull request by rachel-bousfield**
Improves the default values for the intercept hooks

### CODE REVIEW

File: internal/ethapi/api.go [hunk 1]
```diff
@@ -731,10 +731,10 @@ func (s *PublicBlockChainAPI) GetHeaderByHash(ctx context.Context, hash common.H
...
 
 // GetBlockByNumber returns the requested canonical block.
-// * When blockNr is -1 the chain head is returned.
-// * When blockNr is -2 the pending chain head is returned.
-// * When fullTx is true all transactions in the block are returned, otherwise
-//   only the transaction hash is returned.
+//   - When blockNr is -1 the chain head is returned.
+//   - When blockNr is -2 the pending chain head is returned.
+//   - When fullTx is true all transactions in the block are returned, otherwise
+//     only the transaction hash is returned.
```
Comment (PlasmaPower) [inline internal/ethapi/api.go:737]: Let's undo this to avoid upstream diff

File: internal/ethapi/api.go [hunk 2]
```diff
@@ -965,7 +965,7 @@ func DoCall(ctx context.Context, b Backend, args TransactionArgs, blockNrOrHash
 	msg.TxRunMode = txRunMode
 
 	// Arbitrum: support NodeInterface.sol by swapping out the message if needed
-	if core.InterceptRPCMessage != nil {
+	{
```
Comment (PlasmaPower) [inline internal/ethapi/api.go:968]: We can just pull the code out of this block now that there's no longer an if statement

**Review by PlasmaPower — CHANGES_REQUESTED** [review]

File: internal/ethapi/api.go (hunk 1 above)
Comment (RachelFranks) [reply internal/ethapi/api.go]: Oops, didn't realize my editor did this

File: internal/ethapi/api.go (hunk 2 above)
Comment (RachelFranks) [reply internal/ethapi/api.go]: Sure, I was trying to contain the logic in a single scope but it's not necessary

**Review by hkalodner — APPROVED** [review]
LGTM

**Review by PlasmaPower — APPROVED** [review]
LGTM

**Pull request merged**